### ISSUE TITLE
Creating custom user fails in admin

### ISSUE THREAD

**Issue by olmari** [TampereHacklab/mulysa#181]
Admin/Staff can't create new custom user from admin, it ends up in 500 error page, from logs

```
huhti 07 13:57:55 alfa.vaasa.hacklab.fi pipenv[283662]: [2020-04-07 13:57:55,544] 2ef4b732a69e47b494bc6d5c2613d626 ERROR Internal Server Error: /admin/users/customuser/add/
huhti 07 13:57:55 alfa.vaasa.hacklab.fi pipenv[283662]: Traceback (most recent call last):
huhti 07 13:57:55 alfa.vaasa.hacklab.fi pipenv[283662]:   File "/opt/mulysa/.local/share/virtualenvs/mulysa-K2SY2yL-/lib/python3.7/site-packages/django/db/backends/mysql/base.py", line 74, in execute
huhti 07 13:57:55 alfa.vaasa.hacklab.fi pipenv[283662]:     return self.cursor.execute(query, args)
huhti 07 13:57:55 alfa.vaasa.hacklab.fi pipenv[283662]:   File "/opt/mulysa/.local/share/virtualenvs/mulysa-K2SY2yL-/lib/python3.7/site-packages/MySQLdb/cursors.py", line 209, in execute
huhti 07 13:57:55 alfa.vaasa.hacklab.fi pipenv[283662]:     res = self._query(query)
huhti 07 13:57:55 alfa.vaasa.hacklab.fi pipenv[283662]:   File "/opt/mulysa/.local/share/virtualenvs/mulysa-K2SY2yL-/lib/python3.7/site-packages/MySQLdb/cursors.py", line 315, in _query
huhti 07 13:57:55 alfa.vaasa.hacklab.fi pipenv[283662]:     db.query(q)
huhti 07 13:57:55 alfa.vaasa.hacklab.fi pipenv[283662]:   File "/opt/mulysa/.local/share/virtualenvs/mulysa-K2SY2yL-/lib/python3.7/site-packages/MySQLdb/connections.py", line 239, in query
huhti 07 13:57:55 alfa.vaasa.hacklab.fi pipenv[283662]:     _mysql.connection.query(self, query)
huhti 07 13:57:55 alfa.vaasa.hacklab.fi pipenv[283662]: MySQLdb._exceptions.OperationalError: (1048, "Column 'birthday' cannot be null")
huhti 07 13:57:55 alfa.vaasa.hacklab.fi pipenv[283662]: During handling of the above exception, another exception occurred:
huhti 07 13:57:55 alfa.vaasa.hacklab.fi pipenv[283662]: Traceback (most recent call last):
huhti 07 13:57:55 alfa.vaasa.hacklab.fi pipenv[283662]:   File "/opt/mulysa/.local/share/virtualenvs/mulysa-K2SY2yL-/lib/python3.7/site-packages/django/core/handlers/exception.py", line 34, in inner
huhti 07 13:57:55 alfa.vaasa.hacklab.fi pipenv[283662]:     response = get_response(request)
huhti 07 13:57:55 alfa.vaasa.hacklab.fi pipenv[283662]:   File "/opt/mulysa/.local/share/virtualenvs/mulysa-K2SY2yL-/lib/python3.7/site-packages/django/core/handlers/base.py", line 115, in _get_response
huhti 07 13:57:55 alfa.vaasa.hacklab.fi pipenv[283662]:     response = self.process_exception_by_middleware(e, request)
huhti 07 13:57:55 alfa.vaasa.hacklab.fi pipenv[283662]:   File "/opt/mulysa/.local/share/virtualenvs/mulysa-K2SY2yL-/lib/python3.7/site-packages/django/core/handlers/base.py", line 113, in _get_response
huhti 07 13:57:55 alfa.vaasa.hacklab.fi pipenv[283662]:     response = wrapped_callback(request, *callback_args, **callback_kwargs)
huhti 07 13:57:55 alfa.vaasa.hacklab.fi pipenv[283662]:   File "/opt/mulysa/.local/share/virtualenvs/mulysa-K2SY2yL-/lib/python3.7/site-packages/django/contrib/admin/options.py", line 607, in wrapper
huhti 07 13:57:55 alfa.vaasa.hacklab.fi pipenv[283662]:     return self.admin_site.admin_view(view)(*args, **kwargs)
huhti 07 13:57:55 alfa.vaasa.hacklab.fi pipenv[283662]:   File "/opt/mulysa/.local/share/virtualenvs/mulysa-K2SY2yL-/lib/python3.7/site-packages/django/utils/decorators.py", line 130, in _wrapped_view
huhti 07 13:57:55 alfa.vaasa.hacklab.fi pipenv[283662]:     response = view_func(request, *args, **kwargs)
huhti 07 13:57:55 alfa.vaasa.hacklab.fi pipenv[283662]:   File "/opt/mulysa/.local/share/virtualenvs/mulysa-K2SY2yL-/lib/python3.7/site-packages/django/views/decorators/cache.py", line 44, in _wrapped_view_func
huhti 07 13:57:55 alfa.vaasa.hacklab.fi pipenv[283662]:     response = view_func(request, *args, **kwargs)
huhti 07 13:57:55 alfa.vaasa.hacklab.fi pipenv[283662]:   File "/opt/mulysa/.local/share/virtualenvs/mulysa-K2SY2yL-/lib/python3.7/site-packages/django/contrib/admin/sites.py", line 231, in inner
huhti 07 13:57:55 alfa.vaasa.hacklab.fi pipenv[283662]:     return view(request, *args, **kwargs)
huhti 07 13:57:55 alfa.vaasa.hacklab.fi pipenv[283662]:   File "/opt/mulysa/.local/share/virtualenvs/mulysa-K2SY2yL-/lib/python3.7/site-packages/django/utils/decorators.py", line 43, in _wrapper
huhti 07 13:57:55 alfa.vaasa.hacklab.fi pipenv[283662]:     return bound_method(*args, **kwargs)
huhti 07 13:57:55 alfa.vaasa.hacklab.fi pipenv[283662]:   File "/opt/mulysa/.local/share/virtualenvs/mulysa-K2SY2yL-/lib/python3.7/site-packages/django/views/decorators/debug.py", line 76, in sensitive_post_parameters_wrapper
huhti 07 13:57:55 alfa.vaasa.hacklab.fi pipenv[283662]:     return view(request, *args, **kwargs)
huhti 07 13:57:55 alfa.vaasa.hacklab.fi pipenv[283662]:   File "/opt/mulysa/.local/share/virtualenvs/mulysa-K2SY2yL-/lib/python3.7/site-packages/django/utils/decorators.py", line 43, in _wrapper
huhti 07 13:57:55 alfa.vaasa.hacklab.fi pipenv[283662]:     return bound_method(*args, **kwargs)
huhti 07 13:57:55 alfa.vaasa.hacklab.fi pipenv[283662]:   File "/opt/mulysa/.local/share/virtualenvs/mulysa-K2SY2yL-/lib/python3.7/site-packages/django/utils/decorators.py", line 130, in _wrapped_view
huhti 07 13:57:55 alfa.vaasa.hacklab.fi pipenv[283662]:     response = view_func(request, *args, **kwargs)
huhti 07 13:57:55 alfa.vaasa.hacklab.fi pipenv[283662]:   File "/opt/mulysa/.local/share/virtualenvs/mulysa-K2SY2yL-/lib/python3.7/site-packages/django/contrib/auth/admin.py", line 99, in add_view
huhti 07 13:57:55 alfa.vaasa.hacklab.fi pipenv[283662]:     return self._add_view(request, form_url, extra_context)
huhti 07 13:57:55 alfa.vaasa.hacklab.fi pipenv[283662]:   File "/opt/mulysa/.local/share/virtualenvs/mulysa-K2SY2yL-/lib/python3.7/site-packages/django/contrib/auth/admin.py", line 126, in _add_view
huhti 07 13:57:55 alfa.vaasa.hacklab.fi pipenv[283662]:     return super().add_view(request, form_url, extra_context)
huhti 07 13:57:55 alfa.vaasa.hacklab.fi pipenv[283662]:   File "/opt/mulysa/.local/share/virtualenvs/mulysa-K2SY2yL-/lib/python3.7/site-packages/django/contrib/admin/options.py", line 1638, in add_view
huhti 07 13:57:55 alfa.vaasa.hacklab.fi pipenv[283662]:     return self.changeform_view(request, None, form_url, extra_context)
huhti 07 13:57:55 alfa.vaasa.hacklab.fi pipenv[283662]:   File "/opt/mulysa/.local/share/virtualenvs/mulysa-K2SY2yL-/lib/python3.7/site-packages/django/utils/decorators.py", line 43, in _wrapper
huhti 07 13:57:55 alfa.vaasa.hacklab.fi pipenv[283662]:     return bound_method(*args, **kwargs)
huhti 07 13:57:55 alfa.vaasa.hacklab.fi pipenv[283662]:   File "/opt/mulysa/.local/share/virtualenvs/mulysa-K2SY2yL-/lib/python3.7/site-packages/django/utils/decorators.py", line 130, in _wrapped_view
huhti 07 13:57:55 alfa.vaasa.hacklab.fi pipenv[283662]:     response = view_func(request, *args, **kwargs)
huhti 07 13:57:55 alfa.vaasa.hacklab.fi pipenv[283662]:   File "/opt/mulysa/.local/share/virtualenvs/mulysa-K2SY2yL-/lib/python3.7/site-packages/django/contrib/admin/options.py", line 1522, in changeform_view
huhti 07 13:57:55 alfa.vaasa.hacklab.fi pipenv[283662]:     return self._changeform_view(request, object_id, form_url, extra_context)
huhti 07 13:57:55 alfa.vaasa.hacklab.fi pipenv[283662]:   File "/opt/mulysa/.local/share/virtualenvs/mulysa-K2SY2yL-/lib/python3.7/site-packages/django/contrib/admin/options.py", line 1565, in _changeform_view
huhti 07 13:57:55 alfa.vaasa.hacklab.fi pipenv[283662]:     self.save_model(request, new_object, form, not add)
huhti 07 13:57:55 alfa.vaasa.hacklab.fi pipenv[283662]:   File "/opt/mulysa/.local/share/virtualenvs/mulysa-K2SY2yL-/lib/python3.7/site-packages/django/contrib/admin/options.py", line 1081, in save_model
huhti 07 13:57:55 alfa.vaasa.hacklab.fi pipenv[283662]:     obj.save()
huhti 07 13:57:55 alfa.vaasa.hacklab.fi pipenv[283662]:   File "/opt/mulysa/.local/share/virtualenvs/mulysa-K2SY2yL-/lib/python3.7/site-packages/django/contrib/auth/base_user.py", line 66, in save
huhti 07 13:57:55 alfa.vaasa.hacklab.fi pipenv[283662]:     super().save(*args, **kwargs)
huhti 07 13:57:55 alfa.vaasa.hacklab.fi pipenv[283662]:   File "/opt/mulysa/.local/share/virtualenvs/mulysa-K2SY2yL-/lib/python3.7/site-packages/django/db/models/base.py", line 746, in save
huhti 07 13:57:55 alfa.vaasa.hacklab.fi pipenv[283662]:     force_update=force_update, update_fields=update_fields)
huhti 07 13:57:55 alfa.vaasa.hacklab.fi pipenv[283662]:   File "/opt/mulysa/.local/share/virtualenvs/mulysa-K2SY2yL-/lib/python3.7/site-packages/django/db/models/base.py", line 784, in save_base
huhti 07 13:57:55 alfa.vaasa.hacklab.fi pipenv[283662]:     force_update, using, update_fields,
huhti 07 13:57:55 alfa.vaasa.hacklab.fi pipenv[283662]:   File "/opt/mulysa/.local/share/virtualenvs/mulysa-K2SY2yL-/lib/python3.7/site-packages/django/db/models/base.py", line 887, in _save_table
huhti 07 13:57:55 alfa.vaasa.hacklab.fi pipenv[283662]:     results = self._do_insert(cls._base_manager, using, fields, returning_fields, raw)
huhti 07 13:57:55 alfa.vaasa.hacklab.fi pipenv[283662]:   File "/opt/mulysa/.local/share/virtualenvs/mulysa-K2SY2yL-/lib/python3.7/site-packages/django/db/models/base.py", line 926, in _do_insert
huhti 07 13:57:55 alfa.vaasa.hacklab.fi pipenv[283662]:     using=using, raw=raw,
huhti 07 13:57:55 alfa.vaasa.hacklab.fi pipenv[283662]:   File "/opt/mulysa/.local/share/virtualenvs/mulysa-K2SY2yL-/lib/python3.7/site-packages/django/db/models/manager.py", line 82, in manager_method
huhti 07 13:57:55 alfa.vaasa.hacklab.fi pipenv[283662]:     return getattr(self.get_queryset(), name)(*args, **kwargs)
huhti 07 13:57:55 alfa.vaasa.hacklab.fi pipenv[283662]:   File "/opt/mulysa/.local/share/virtualenvs/mulysa-K2SY2yL-/lib/python3.7/site-packages/django/db/models/query.py", line 1204, in _insert
huhti 07 13:57:55 alfa.vaasa.hacklab.fi pipenv[283662]:     return query.get_compiler(using=using).execute_sql(returning_fields)
huhti 07 13:57:55 alfa.vaasa.hacklab.fi pipenv[283662]:   File "/opt/mulysa/.local/share/virtualenvs/mulysa-K2SY2yL-/lib/python3.7/site-packages/django/db/models/sql/compiler.py", line 1391, in execute_sql
huhti 07 13:57:55 alfa.vaasa.hacklab.fi pipenv[283662]:     cursor.execute(sql, params)
huhti 07 13:57:55 alfa.vaasa.hacklab.fi pipenv[283662]:   File "/opt/mulysa/.local/share/virtualenvs/mulysa-K2SY2yL-/lib/python3.7/site-packages/django/db/backends/utils.py", line 68, in execute
huhti 07 13:57:55 alfa.vaasa.hacklab.fi pipenv[283662]:     return self._execute_with_wrappers(sql, params, many=False, executor=self._execute)
huhti 07 13:57:55 alfa.vaasa.hacklab.fi pipenv[283662]:   File "/opt/mulysa/.local/share/virtualenvs/mulysa-K2SY2yL-/lib/python3.7/site-packages/django/db/backends/utils.py", line 77, in _execute_with_wrappers
huhti 07 13:57:55 alfa.vaasa.hacklab.fi pipenv[283662]:     return executor(sql, params, many, context)
huhti 07 13:57:55 alfa.vaasa.hacklab.fi pipenv[283662]:   File "/opt/mulysa/.local/share/virtualenvs/mulysa-K2SY2yL-/lib/python3.7/site-packages/django/db/backends/utils.py", line 86, in _execute
huhti 07 13:57:55 alfa.vaasa.hacklab.fi pipenv[283662]:     return self.cursor.execute(sql, params)
huhti 07 13:57:55 alfa.vaasa.hacklab.fi pipenv[283662]:   File "/opt/mulysa/.local/share/virtualenvs/mulysa-K2SY2yL-/lib/python3.7/site-packages/django/db/backends/mysql/base.py", line 79, in execute
huhti 07 13:57:55 alfa.vaasa.hacklab.fi pipenv[283662]:     raise utils.IntegrityError(*tuple(e.args))
huhti 07 13:57:55 alfa.vaasa.hacklab.fi pipenv[283662]: django.db.utils.IntegrityError: (1048, "Column 'birthday' cannot be null")
```

**Comment by olmari** [comment]
It seems that this page model needs to create some default values and then continue, or something. 
![Custom User add in Admin UI](https://user-images.githubusercontent.com/5422161/78668588-97eca880-78e3-11ea-9daa-152f97ce4478.png)

**Comment by olmari** [comment]
Fixed in 37f39ce79dbbe4b1f4de4d3984e381c35212b00a
